api/application_submissions/index.js

```diff
@@ -31,24 +31,31 @@ router.post('/', async (req, res) => {
   }
 });
 
-router.put('/:id', async (req, res) => {
-  const { id } = req.params;
+router.put('/', async (req, res) => {
+  const { ids } = req.body; 
   const { decision } = req.body;
 
   try {
-    let applicationSubmission = await ApplicationSubmission.findById(id);
-    if (applicationSubmission) {
-      if (decision === 'ACCEPTED') {
-        const { skilled_impact_request_id } = applicationSubmission;
-        applicationSubmission = await ApplicationSubmission.acceptAndDenyAllOthers(id, skilled_impact_request_id);
-      } else {
-        applicationSubmission = await ApplicationSubmission.update(id, decision);
-      }
-      res.status(200).json({ applicationSubmission, message: 'Submission updated in database' });
-    } else {
-      res.status(404).json({ message: 'Submission not found in database' });
+    const applicationSubmissions = await ApplicationSubmission.updateAll(ids, decision);
+    // return 200 if all submissions successfully updated
+    if(applicationSubmissions.length == ids.length) {
+      res.status(200).json({ applicationSubmissions, message: 'Submissions updated in database' });
     }
-  } catch (error) {
+    // return 404 if no submissions found, else return 202 (accepted, but some processing still needed)
```

Comment:
I think the 404 is pretty obvious to the reader, but can you move the 202 comment down to the line where we actually return 202?